### PR TITLE
Portable youtube-dl 2021.12.17

### PR DESCRIPTION
**Pull request by LeonarddeR**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/68800)

### CODE REVIEW

**Comment by leonardder** [comment]
I guess the pipeline is not yet ready for portables?

File: manifests/y/youtube-dl/youtube-dl/2021.12.17/youtube-dl.youtube-dl.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+Commands:
+- youtube-dl
+ReleaseDate: 2021-12-17
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/ytdl-org/youtube-dl/releases/download/2021.12.17/youtube-dl.exe
+  InstallerSha256: 26E5C00C35C5C3EDC86DFC0A720AED109A13B1B7C67AC654A0CE8FF82A1F2C16
+  AppsAndFeaturesEntries:
+  - DisplayName: youtube-dl
+    DisplayVersion: 2021.12.17
```
Comment (Trenly) [inline manifests/y/youtube-dl/youtube-dl/2021.12.17/youtube-dl.youtube-dl.installer.yaml:17]:
```suggestion
```
It is not recommended to have a display version which is the same as the package version

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by Trenly** [comment]
> I guess the pipeline is not yet ready for portables?

https://devblogs.microsoft.com/commandline/windows-package-manager-1-3/#submitting-manifests-for-portable-packages

File: manifests/y/youtube-dl/youtube-dl/2021.12.17/youtube-dl.youtube-dl.installer.yaml (hunk 1 above)
Comment (leonardder) [reply manifests/y/youtube-dl/youtube-dl/2021.12.17/youtube-dl.youtube-dl.installer.yaml:17]: Could you elaborate? Something like this?
```suggestion
    DisplayVersion: 2021.12.17.0
```

In that case, the display version is equal to the file version of the executable.

**Review by ItzLevvie — APPROVED** [review]

**Comment by leonardder** [comment]
Everything should be OK.

**Pull request merged**